mage_ai/data_cleaner/statistics/calculator.py

```diff
@@ -205,13 +202,25 @@ def __statistics_overview(self, series, col):
                         .iloc[:OUTLIER_SAMPLE_COUNT]
                         .tolist()
                     )
-            elif column_type == DATETIME:
+            elif column_type == ColumnType.DATETIME:
                 dates = pd.to_datetime(series_non_null, utc=True, errors='coerce').dropna()
                 data[f'{col}/max'] = dates.max().isoformat()
                 data[f'{col}/median'] = (
                     dates.sort_values().iloc[math.floor(len(dates) / 2)].isoformat()
                 )
                 data[f'{col}/min'] = dates.min().isoformat()
+            elif column_type == ColumnType.TEXT:
+                text_series = series_non_null
+                data[f'{col}/avg_str_length'] = text_series.apply(lambda string: len(string)).mean()
+                text_series = text_series.str.split(' ')
+                data[f'{col}/avg_word_count'] = text_series.apply(lambda words: len(words)).mean()
+                text_series = text_series.explode()
+                text_series = text_series[~text_series.str.match('\s*')]
+                data[f'{col}/word_distribution'] = (
+                    text_series.value_counts().head(VALUE_COUNT_LIMIT).todict()
```

Comment:
@skunichetty This line throws error.

Reply:
This code wasn't even supposed to be in this branch, that's my bad. I can put out a really quick fix for it.

Reply:
Actually, I have another branch that has this fixed, i'll just pull that branch instead.

Reply:
Let's remove the bad change first.

Reply:
Fixed at PR here: https://github.com/mage-ai/mage-ai/pull/332